logging/google/cloud/logging/_http.py

```diff
@@ -428,7 +428,7 @@ def metric_update(self, project, metric_name, filter_, description):
             'filter': filter_,
             'description': description,
         }
-        self.api_request(method='PUT', path=target, data=data)
+        return self.api_request(method='PUT', path=target, data=data)
```

Comment:
Ditto here on an update change I'm not sure I should make / probably shouldn't be in this PR